profiles/common/modules/custom/tmgmt_poetry/templates/tmgmt-poetry-progress-field.tpl.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @file
+ * Template file for the TMGMT POETRY progress field.
+ *
+ * Available custom variables:
+ * - $stats: An array with statistics data.
+ * - $title: String which can be used as a tooltip.
+ */
+drupal_add_css(drupal_get_path('module', 'tmgmt_poetry') . '/css/tmgmt_poetry.css');
```

Comment:
`drupal_add_css()` should be called into this theme hook preprocess function, something like:
```php
function template_preprocess_tmgmt_poetry_progress_field(&$variables) {
  drupal_add_css(drupal_get_path('module', 'tmgmt_poetry') . '/css/tmgmt_poetry.css');
}
```